### PR TITLE
When rendering stuff outside of ON OBJECT RENDER hooks, use immediate mode

### PR DESCRIPTION
**Pull request by The-E**

### CODE REVIEW

**Review by EatThePath — APPROVED** [review]
Looks solid, restores old behavior from before #4664 when not in the object render hook, and resolves a regression stemming from that change.

I think it could benefit from a sprinkle of explanatory comments for future tinkerers, but I wouldn't make that a blocking request

**Review by EatThePath — APPROVED** [review]
comments look good

**Review by Goober5000 — CHANGES_REQUESTED** [review]
The same fix should be applied to `drawModelOOR`, immediately below it, since these are twin functions.

Also, if these two script functions are really intended to be used outside On Object Render, and we intend to support that functionality, then the function header comments should be updated.

**Comment by The-E** [comment]
> The same fix should be applied to `drawModelOOR`, immediately below it, since these are twin functions.
> 
> Also, if these two script functions are really intended to be used outside On Object Render, and we intend to support that functionality, then the function header comments should be updated.

Done

**Comment by Goober5000** [comment]
Good, but regarding the documentation - if our intent is that scripters can use this anywhere, then the "Use with extreme care" sentence should be removed from both functions' headers.

**Comment by The-E** [comment]
> Good, but regarding the documentation - if our intent is that scripters can use this anywhere, then the "Use with extreme care" sentence should be removed from both functions' headers.

done

**Review by Goober5000 — APPROVED** [review]
Looks good now, although the previous multi-string single-arg description was turned into a multi-arg description

File: code/scripting/api/libs/graphics.cpp [hunk 1]
```diff
@@ -856,8 +856,8 @@ ADE_FUNC(draw3dLine, l_Graphics, "vector origin, vector destination, [boolean tr
 
 // Aardwolf's test code to render a model, supposed to emulate WMC's gr.drawModel function
 ADE_FUNC(drawModel, l_Graphics, "model model, vector position, orientation orientation",
-         "Draws the given model with the specified position and orientation - Use with extreme care, may not work "
-         "properly in all scripting hooks.  Note: this method does NOT use CurrentResizeMode.",
+         "Draws the given model with the specified position and orientation",
+	     "Note: this method does NOT use CurrentResizeMode.",
```
Comment (Goober5000) [inline code/scripting/api/libs/graphics.cpp:860]:
```suggestion
         "Draws the given model with the specified position and orientation.  "
	     "Note: this method does NOT use CurrentResizeMode.",
```

**Pull request merged**